ts/parts/Axis.ts

```diff
@@ -4791,12 +4791,13 @@ class Axis implements AxisComposition {
                 });
 
             } else if (
+                axis.dateTime &&
                 axis.isDatetimeAxis &&
```

Comment:
Instead of the `isDatetimeAxis` flag, can we simply test for `axis.dateTime`? That means we also have to delete `axis.dateTime` when updating.

Reply:
It is different in the sense that it indicates, if the axis type is set to `datetime`, not if the axis is capable of date/time support like the composition does.

Reply:
Data grouping uses the composition functions without checking for `isDateTimeAxis`. If that can be solved somehow, we can remove `isDateTimeAxis`.

Reply:
I believe, I have figured it out.